admin/static/js/sales_analytics/sa-user-session.js

```diff
@@ -0,0 +1,212 @@
+'use strict';
+
+var keen = require('keen-js');
+var ss = require('simple-statistics');
+
+var SalesAnalytics = function() {
+    var self = this;
+
+    self.keenClient = new keen({
+        projectId: keenProjectId,
+        readKey : keenReadKey
+    });
+
+    self.keenFilters = {
+        nullUserFilter: {
+            property_name: 'user.id',
+            operator: 'ne',
+            property_value: null
+        },
+        inactiveUserFilter: {
+            property_name: 'user.id',
+            operator: 'ne',
+            property_value: ''
+        }
+    };
```

Comment:
Again looks like quite a bit of repetition between files. Please consider the DRY principle.